src/main/java/com/bio4j/model/NCBITaxonomyGraph.java

```diff
@@ -65,46 +67,66 @@
 
       Values are, among others, *species*, *genus*, or *no rank*; see the enum below.
     */
-    public final TaxonomicRank taxonomicRank = new TaxonomicRank();
-    public final class TaxonomicRank extends Property<TaxonomicRanks> implements FromAny, ToOne {
+    public final Rank rank = new Rank();
+    public final class Rank extends Property<TaxonomicRank> implements FromAny, ToOne {
 
-      private TaxonomicRank() { super(TaxonomicRanks.class); }
+      private Rank() { super(TaxonomicRank.class); }
```

Comment:
Motivation for this change is that the way to refer to the property is `g.taxon.taxonomicRank` which is repetitive. `g.taxon.rank` is shorter and unambiguous.